src/components/flash/flash.js

```diff
@@ -305,19 +318,19 @@ class Flash extends React.Component {
           key={ title }
           title={ title }
           open={ this.state.dialogs[title] || false }
-          onCancel={ this.toggleDialog.bind(this, title) }
+          onCancel={ this.toggleDialog(title) }
         >
           { desc }
         </Alert>
       );
 
       // create text for item
-      text = (
+      return (
```

Comment:
Currently I've implemented the second option but happy to switch it if preferred.